pandas/tests/io/test_parquet.py

```diff
@@ -564,6 +564,13 @@ def test_additional_extension_types(self, pa):
         )
         check_round_trip(df, pa)
 
+    @td.skip_if_no("pyarrow", min_version="0.14")
+    def test_timestamp_nanoseconds(self, pa):
+        # with version 2.0, pyarrow defaults to writing the nanoseconds, so
+        # this should work with error
```

Comment:
Should this say "work **without** error"?

Reply:
yes, of course ... !